examples/benchmark/scene_test_2.py

```diff
@@ -0,0 +1,189 @@
+# -*- coding: utf-8 -*-
+# vispy: testskip
+# -----------------------------------------------------------------------------
+# Copyright (c) 2014, Vispy Development Team.
+# Distributed under the (new) BSD License. See LICENSE.txt for more info.
+# -----------------------------------------------------------------------------
+"""
+Compare an optimal plot grid implementation to the same functionality
+provided by scenegraph.
```

Comment:
It works for me. I guess this is a backend issue--you are getting a call to `on_draw` before `__init__` has completed.